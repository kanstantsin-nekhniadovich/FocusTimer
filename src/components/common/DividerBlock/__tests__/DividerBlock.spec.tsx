import React from 'react';
import { render } from '@testing-library/react-native';

import { DividerBlock } from '..';

const HEIGHT = 100;

describe('<DividerBlock />', () => {
  it('test DividerBlock snapshot', () => {
    const tree = render(<DividerBlock />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should render DividerBlock with passed height in props', () => {
    const { container } = render(<DividerBlock height={HEIGHT} />);

    expect(container.props.height).toEqual(HEIGHT);
  });
});
