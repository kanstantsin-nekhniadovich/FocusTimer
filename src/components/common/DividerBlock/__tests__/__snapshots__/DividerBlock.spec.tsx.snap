// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DividerBlock /> test DividerBlock snapshot 1`] = `
<View
  style={
    Object {
      "height": 20,
      "width": "100%",
    }
  }
/>
`;
